sechub-cli/src/mercedes-benz.com/sechub/util/errorhandler.go

```diff
@@ -9,27 +9,27 @@ import (
 	"os"
 )
 
-// HandleHTTPError handler method for http errors
-func HandleHTTPError(err error, exitcode int) {
+// HandleError - handler method for common errors
+func HandleError(err error, exitcode int) {
 	if err != nil {
-		LogError(fmt.Sprintf("The HTTP request failed with error '%s'", err))
+		LogError(fmt.Sprintf("Error: %s\n", err))
 		os.Exit(exitcode)
 	}
 }
 
-// HandleError handler method for common errors
-func HandleError(err error, exitcode int) {
+// HandleHTTPError - handler method for http errors
+func HandleHTTPError(err error, exitcode int) {
 	if err != nil {
-		LogError(fmt.Sprintf("Error: %s\n", err))
+		LogError(fmt.Sprintf("The HTTP request failed with error '%s'", err))
```

Comment:
Please clarify, the "error" is the HTTP response code or the program error?

Reply:
The Github generated diffs are a bit confusing.
I switched only the positions of `HandleHTTPError` and `HandleError` so the more general function comes first. (no code changes)

See the complete code snippet:
```go
// HandleError - handler method for common errors
func HandleError(err error, exitCode int) {
	if err != nil {
		LogError(fmt.Sprintf("Error: %s\n", err))
		os.Exit(exitCode)
	}
}

// HandleHTTPError - handler method for http errors
func HandleHTTPError(err error, exitCode int) {
	if err != nil {
		LogError(fmt.Sprintf("The HTTP request failed with error '%s'", err))
		os.Exit(exitCode)
	}
}
```
I think, the output of both is fine.